test/helpers/cilium.go

```diff
@@ -658,6 +658,7 @@ func (s *SSHMeta) ValidateNoErrorsInLogs(duration time.Duration) {
 	}()
 
 	blacklist := GetBadLogMessages()
+	// TODO: We need to print the error messages.
```

Comment:
We should open a good-first-issue issue instead for this.

Reply:
https://github.com/cilium/cilium/issues/18159